docs/pages/cldimage/configuration.mdx

```diff
@@ -1933,13 +1939,26 @@ until it's available, then force refresh the image instance in the DOM.
 **Examples**
 
 ```jsx copy
-onError={() => {
+onError={(e) => {
```

Comment:
does this actually pass an error/event? i thought this just passed back the element itself

Reply:
Actually it does pass event object, this event has a property called target which contains the pointer to element.

Reply:
ah just double checked in my code and you're right, for some reason though it only passed the element. thanks for confirming!